### PR TITLE
Add WebAssembly.validate

### LINKED ISSUE #354: Implement WebAssembly.validate
> The WebAssembly.validate() function validates a given typed array of WebAssembly binary code, returning whether the bytes form a valid wasm module (true) or not (false).

```js
WebAssembly.validate(bufferSource)
```

### PR DESCRIPTION
**Pull request by chicoxyzzy**
According to https://webassembly.github.io/spec/js-api/#dom-webassembly-validate

Not sure if my test templates even work 🙃 How to use these `.tjs` properly?

closes #354

### CODE REVIEW

File: packages/webassemblyjs/test/compile/fixtures/valide-module/exec.tjs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+it("should return return true on WebAssembly.validate", () => {
+  return WebAssembly.validate(wasmmodule).then(m => {
```
Comment (xtuc) [inline packages/webassemblyjs/test/compile/fixtures/valide-module/exec.tjs]: Validate doesn't return a promise but a boolean.

Comment (chicoxyzzy) [reply packages/webassemblyjs/test/compile/fixtures/valide-module/exec.tjs]: Oops, yes. So tjs doesn't work :)

Comment (xtuc) [reply packages/webassemblyjs/test/compile/fixtures/valide-module/exec.tjs]: No worries, I'm currently fixiing it locally.

Comment (xtuc) [reply packages/webassemblyjs/test/compile/fixtures/valide-module/exec.tjs]: Actually tjs is just a regular JavaScript file, which is executed by `make test`.

Comment (chicoxyzzy) [reply packages/webassemblyjs/test/compile/fixtures/valide-module/exec.tjs]: Got it, thanks! 👍

**Pull request merged**